charts/argocd-certificate-refresh/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for argocd-certificate-refresh.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# docker image information
+image: quay.io/devtron/argocd-cert-refresh:v1
+
+
+# values for cronjob
+cronjob:
+  schedule: "*/5 * * * *"
```

Comment:
why is the default schedule set for every 5 mins?

Reply:
Set it for every midnight every 6 months

Reply:
Done changed in schedule time